raiden/transfer/mediated_transfer/events.py

```diff
@@ -796,7 +800,7 @@ def from_dict(cls, data: Dict[str, Any]) -> "SendSecretRequestLight":
             expiration=BlockExpiration(int(data["expiration"])),
             secrethash=deserialize_secret_hash(data["secrethash"]),
             signed_secret_request=data["signed_secret_request"],
-            sender=None
+            sender=data["sender"]
```

Comment:
We should use           `  sender=to_canonical_address(data["sender"]),` here to create a normalized address of type bytes